src/System.Private.CoreLib/src/System/StubHelpers.cs

```diff
@@ -786,7 +786,7 @@ internal static unsafe void ConvertContentsToManaged_Type(ref System.Type[] mana
             }
         }
 
-        internal static unsafe void ConvertContentsToManaged_Exception(ref Exception[] managedArray, IntPtr pNativeHome)
+        internal static unsafe void ConvertContentsToManaged_Exception(ref Exception?[] managedArray, IntPtr pNativeHome)
```

Comment:
Can't these `managedArray` parameters in `ConvertContentsToManaged_Exception` and ConvertContentsToManaged_Type` be nullable?

Reply:
Yup.  Fixed.  It doesn't end up mattering as the code that calls them is emitted IL rather than C#/